span.go

```diff
@@ -87,6 +87,14 @@ type Span interface {
 	// See the `SetBaggageItem` notes about `restrictedKey`.
 	BaggageItem(restrictedKey string) string
 
+	// ForeachBaggageItem allows reading all baggage items stored in the span.
```

Comment:
Is it worth documenting whether ForeachBaggageItem has a stable iteration order, or if the order can differ between consecutive calls of ForeachBaggageItem?